Source/Section/PDFSectionObject.swift

```diff
@@ -0,0 +1,151 @@
+//
+//  PDFSectionObject.swift
+//  TPPDF
+//
+//  Created by Marco Betschart on 05.05.18.
+//
+
+class PDFSectionObject: PDFObject {
+
+	var section: PDFSection
+
+	init(section: PDFSection) {
+		self.section = section
+	}
+
+	override func calculate(generator: PDFGenerator, container: PDFContainer) throws -> [(PDFContainer, PDFObject)] {
+		var result: [(PDFContainer, PDFObject)] = []
+
+		let originalLeftIndent = [
+			PDFContainer.contentLeft: generator.layout.indentation.leftIn(container: .contentLeft),
+			PDFContainer.contentCenter: generator.layout.indentation.leftIn(container: .contentCenter),
+			PDFContainer.contentRight: generator.layout.indentation.leftIn(container: .contentRight)
+		]
+
+		let originalRightIndent = [
+			PDFContainer.contentLeft: generator.layout.indentation.rightIn(container: .contentLeft),
+			PDFContainer.contentCenter: generator.layout.indentation.rightIn(container: .contentCenter),
+			PDFContainer.contentRight: generator.layout.indentation.rightIn(container: .contentRight)
+		]
+
```

Comment:
`generator.layout.indentation` is instance of struct `PDFLayoutIndentations`. It should be possible to store original left and right indentation state simply by doing storing it in a local instance, as structs are[ always copied ](https://developer.apple.com/library/content/documentation/Swift/Conceptual/Swift_Programming_Language/ClassesAndStructures.html) and not referenced.

```swift
let originalIndent = generator.layout.indentation.content
```
and afterwards reassign it:
```swift
generator.layout.indentation.content = originalIndent
```

Reply:
good catch! Fixed.